interface_specifications/pe_openapi/apis/v1/d1_pe_definition_statuses_put.md

```diff
@@ -0,0 +1,38 @@
+### Status Notification to Verifier
+
+`PUT <verifier-example.io>pe/definitions/{definition_id}/statuses`
+
+
+#### Description
+
+This is an optional interaction and hence party can choose not to implement it. The PE REST Backend invokes an HTTP REST API exposed by the party to notify about the status of exchange.
+
+#### Parameters
+
+Path Variables:
+* `definition_id`: definition id e.g. `32f54163-7166-48f1-93d8-ff217bdb0653`
+
+Request Body
+
+```json
+{
+  "thread":{
```

Comment:
By the looks of creating the definitions, the thread_id is bound to the newly created definition id. Meaning we don't need to supply the thread id perse in the request right?

Reply:
Currently it may seem that ThreadID is completely determinable from Presentation Definition ID. For now, since, for one thread, we expect only one presentation definition and one presentation. It will not be true in future where on one thread new presentation definitions can be 'proposed' and when multiple presentations are possible.

 It will be a good thing to still pass it. But for current case it is removable. So, shall I keep it?

Reply:
For one thread having multiple definitions, my comment still holds right?

As long as we have the thread id stored next to the definition ids, we only need an actual definition id passed in to get the accompanying thread id from persistence

Reply:
fixed.